From 68397ac09343d8af9ad971a56d57651ea4a2ccf0 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Thu, 9 May 2019 16:24:58 +0200
Subject: [PATCH] TRITON-1646 Add flexible_disk package to dev-sample-data
 Reviewed by: Mike Gerdts <mike.gerdts@joyent.com> Approved by: Mike Gerdts
 <mike.gerdts@joyent.com>

---
 etc/sample-packages.json | 63 ++++++++++++++++++++++++++++++++++++++++
 1 file changed, 63 insertions(+)

diff --git a/etc/sample-packages.json b/etc/sample-packages.json
index 8dd4e62..3c8cc16 100644
--- a/etc/sample-packages.json
+++ b/etc/sample-packages.json
@@ -161,5 +161,68 @@
     "default": false,
     "v": 1,
     "vcpus": 1
+  },
+  {
+    "name": "sample-bhyve-flexible-1G",
+    "version": "1.0.0",
+    "active": true,
+    "max_lwps": 4000,
+    "max_physical_memory": 1024,
+    "max_swap": 4096,
+    "quota": 24576,
+    "zfs_io_priority": 64,
+    "fss": 64,
+    "cpu_cap": 100,
+    "billing_tag": "sample-1G-bhyve",
+    "default": true,
+    "v": 1,
+    "vcpus": 1,
+    "brand": "bhyve",
+    "flexible_disk": true
+  },
+  {
+    "name": "sample-bhyve-reserved-snapshots-space",
+    "version": "1.0.0",
+    "active": true,
+    "max_lwps": 4000,
+    "max_physical_memory": 1024,
+    "max_swap": 4096,
+    "quota": 40960,
+    "zfs_io_priority": 64,
+    "fss": 64,
+    "cpu_cap": 100,
+    "billing_tag": "sample-1G-bhyve",
+    "default": true,
+    "v": 1,
+    "vcpus": 1,
+    "brand": "bhyve",
+    "flexible_disk": true,
+    "disks": [
+            { },
+            { "size": 10240 }
+    ]
+  },
+  {
+    "name": "sample-bhyve-three-disks",
+    "version": "1.0.0",
+    "active": true,
+    "max_lwps": 4000,
+    "max_physical_memory": 1024,
+    "max_swap": 4096,
+    "quota": 24576,
+    "zfs_io_priority": 64,
+    "fss": 64,
+    "cpu_cap": 100,
+    "billing_tag": "sample-1G-bhyve",
+    "default": true,
+    "v": 1,
+    "vcpus": 1,
+    "brand": "bhyve",
+    "flexible_disk": true,
+    "disks": [
+        { },
+        { "size": 6144 },
+        { "size": "remaining"}
+    ]
   }
 ]
-- 
2.21.0

